source/about/mattermost-v10-changelog.md

```diff
@@ -102,7 +102,7 @@ New setting options were added to ``config.json``. Below is a list of the additi
  - Updated minimum Safari version to 17.4+ and minimum Firefox version to 119+.
 
 ### Important Upgrade Notes
- - Both Mattermost v10.5.0 and v10.5.1 come bundled with a Jira plugin version that contains a bug causing the plugin configuration to disappear. We strongly discourage upgrading to these versions to avoid this issue. If you have already upgraded to 10.5.0 or 10.5.1, we recommend downgrading the Jira plugin to a stable version.
+ - Both Mattermost v10.5.0 and v10.5.1 come bundled with a Jira plugin version that contains a bug causing the plugin configuration to disappear. We strongly discourage upgrading to these versions to avoid this issue. If you have already upgraded to 10.5.0 or 10.5.1, we recommend upgrading the Jira version to v4.2.1 or ideally upgrading both Mattermost and Jira to the latest version.
```

Comment:
```suggestion
 - Mattermost versions v10.5.0 and v10.5.1 include a bundled Jira plugin (v4.2.0) that contains a bug which may cause plugin configuration settings to disappear. We strongly advise against upgrading to these versions to avoid potential disruption.  
If you have already upgraded to v10.5.0 or v10.5.1, we recommend updating the Jira plugin to version v4.2.1, or preferably, upgrading both Mattermost and the Jira plugin to their latest available versions.
```